rasa/core/channels/socketio.py

```diff
@@ -163,6 +165,8 @@ def __init__(
         self.namespace = namespace
         self.socketio_path = socketio_path
         self.sio = None
+        self.metadata_key = metadata_key
+        self.environ = None
```

Comment:
Why is this new attribute necessary?

Reply:
Removing `environ`. (I was looking at adding an option to channel to include select environment values that could be useful as metadata.)